### PR TITLE
[docs] Update docs for VSO release 0.4.3

### PR DESCRIPTION
**Pull request by kschoche**
Update API docs, helm docs.

### CODE REVIEW

File: website/content/docs/platform/k8s/vso/api-reference.mdx [hunk 1]
```diff
@@ -462,9 +462,9 @@ _Appears in:_
...
-| `path` _string_ | Path in Vault to get the credentials for, and is relative to Mount. Please consult [secrets](/vault/docs/secrets) if you are uncertain about what 'path' should be set to. |
-| `params` _object (keys:string, values:string)_ | Params that can be passed when requesting credentials/secrets. When Params is set the configured RequestHTTPMethod will be ignored. See RequestHTTPMethod for more details. Please consult [secrets](/vault/docs/secrets) if you are uncertain about what 'params' should/can be set to. |
+| `requestHTTPMethod` _string_ | RequestHTTPMethod to use when syncing Secrets from Vault. Setting a value here is not typically required. If left unset the Operator will make requests using the GET method. In the case where Params are specified the Operator will use the PUT method. Please consult [docs](/vault/docs/secrets) if you are uncertain about what method to use. Of note, the Vault client treats PUT and POST as being equivalent. The underlying Vault client implementation will always use the PUT method. |
```
Comment (benashz) [inline website/content/docs/platform/k8s/vso/api-reference.mdx:465]: Is there a reason we are changing the link names here? My guess is that there would be no change needed here.

File: website/content/docs/platform/k8s/vso/api-reference.mdx [hunk 2]
```diff
@@ -462,9 +462,9 @@ _Appears in:_
...
+| `path` _string_ | Path in Vault to get the credentials for, and is relative to Mount. Please consult [docs](/vault/docs/secrets) if you are uncertain about what 'path' should be set to. |
```
Comment (benashz) [inline website/content/docs/platform/k8s/vso/api-reference.mdx:466]: Similar comment as above.

**Review by benashz — COMMENTED** [review]
Are the changes to the api-reference docs necessary? I would guess that we only need to ensure that the links are relative.

File: website/content/docs/platform/k8s/vso/api-reference.mdx (hunk 1 above)
Comment (kschoche) [reply website/content/docs/platform/k8s/vso/api-reference.mdx:465]: Yeah very good point, as I'm reviewing the auto-generated bits I'm thinking I'll back them out. Good call.

**Review by benashz — APPROVED** [review]
LGTM!

**Pull request merged**